sdk/appplatform/mgmt/src/main/java/com/azure/resourcemanager/appplatform/implementation/SpringAppImpl.java

```diff
@@ -0,0 +1,282 @@
+import com.azure.resourcemanager.appplatform.models.AppResourceProperties;
+import com.azure.resourcemanager.appplatform.models.ManagedIdentityProperties;
+import com.azure.resourcemanager.appplatform.models.PersistentDisk;
+import com.azure.resourcemanager.appplatform.models.ResourceUploadDefinition;
+import com.azure.resourcemanager.appplatform.models.SpringApp;
+import com.azure.resourcemanager.appplatform.models.SpringAppDeployment;
+import com.azure.resourcemanager.appplatform.models.SpringAppDeployments;
+import com.azure.resourcemanager.appplatform.models.SpringService;
+import com.azure.resourcemanager.appplatform.models.TemporaryDisk;
+import com.azure.resourcemanager.appplatform.models.UserSourceType;
+import com.azure.resourcemanager.resources.fluentcore.arm.models.implementation.ExternalChildResourceImpl;
+import com.azure.resourcemanager.resources.fluentcore.model.Creatable;
+import reactor.core.publisher.Mono;
+
+import java.io.File;
+import java.time.OffsetDateTime;
+
+public class SpringAppImpl
+    extends ExternalChildResourceImpl<SpringApp, AppResourceInner, SpringServiceImpl, SpringService>
+    implements SpringApp, SpringApp.Definition, SpringApp.Update {
+    private Creatable<SpringAppDeployment> springAppDeploymentToCreate = null;
+
+    SpringAppImpl(String name, SpringServiceImpl parent, AppResourceInner innerObject) {
+        super(name, parent, innerObject);
+    }
+
+    @Override
+    public boolean isPublic() {
+        if (inner().properties() == null) {
+            return false;
+        }
+        return inner().properties().publicProperty();
+    }
+
+    @Override
+    public boolean isHttpsOnly() {
+        if (inner().properties() == null) {
+            return false;
+        }
+        return inner().properties().httpsOnly();
+    }
+
+    @Override
+    public String url() {
+        if (inner().properties() == null) {
+            return null;
+        }
+        return inner().properties().url();
+    }
+
+    @Override
+    public TemporaryDisk temporaryDisk() {
+        if (inner().properties() == null) {
+            return null;
+        }
+        return inner().properties().temporaryDisk();
+    }
+
+    @Override
+    public PersistentDisk persistentDisk() {
+        if (inner().properties() == null) {
+            return null;
+        }
+        return inner().properties().persistentDisk();
+    }
+
+    @Override
+    public ManagedIdentityProperties identity() {
+        return inner().identity();
+    }
+
+    @Override
+    public OffsetDateTime createdTime() {
+        if (inner().properties() == null) {
+            return null;
+        }
+        return inner().properties().createdTime();
+    }
+
+    @Override
+    public String activeDeployment() {
+        if (inner().properties() == null) {
+            return null;
+        }
+        return inner().properties().activeDeploymentName();
+    }
+
+    @Override
+    public SpringAppDeployments deploy() {
```

Comment:
`deployments` would be a better name? Current you will have `deploy().deleteByNameAsync` which seems not clear.